client/src/Pos/Client/Txp/Util.hs

```diff
@@ -142,18 +151,30 @@ instance Buildable TxError where
         bprint "Redemption address balance is 0"
     build (SafeSignerNotFound addr) =
         bprint ("Address "%build%" has no associated safe signer") addr
+    build SignedTxNotBase16Format =
+        "Externally-signed transaction is not in Base16-format."
+    build (SignedTxUnableToDecode msg) =
+        bprint ("Unable to decode externally-signed transaction: "%stext) msg
+    build SignedTxSignatureNotBase16Format =
+        "Signature of externally-signed transaction is not in Base16-format."
+    build (SignedTxInvalidSignature msg) =
+        bprint ("Signature of externally-signed transaction is invalid: "%stext) msg
     build (GeneralTxError msg) =
         bprint ("Transaction creation error: "%stext) msg
 
 isCheckedTxError :: TxError -> Bool
 isCheckedTxError = \case
-    NotEnoughMoney{}        -> True
-    NotEnoughAllowedMoney{} -> True
-    FailedToStabilize{}     -> False
-    OutputIsRedeem{}        -> True
-    RedemptionDepleted{}    -> True
-    SafeSignerNotFound{}    -> True
-    GeneralTxError{}        -> True
+    NotEnoughMoney{}                   -> True
+    NotEnoughAllowedMoney{}            -> True
+    FailedToStabilize{}                -> False
+    OutputIsRedeem{}                   -> True
+    RedemptionDepleted{}               -> True
+    SafeSignerNotFound{}               -> True
+    SignedTxNotBase16Format{}          -> True
+    SignedTxUnableToDecode{}           -> True
+    SignedTxSignatureNotBase16Format{} -> True
+    SignedTxInvalidSignature{}         -> True
+    GeneralTxError{}                   -> True
```

Comment:
Its a really bad idea to include semanitcally null changes (like whitespace) with real changes (especially in large PRs) because it make it difficult for the reviewer.